app/konnect/api/portal-auth/portal-rbac-guide.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Portal RBAC Setup
+content-type: tutorial
+---
+
+## Overview
+
+Role-based Access Control (RBAC) allows you to apply `API Viewer` and `API Consumer` roles in order to manage how developers can access services published to the dev portal. 
+
+* API Viewer: provides read access to the documentation associated with services. 
+* API Consumer: can register applications to consume services. 
+
+You can use the API to create a team of developers, assign a role to the team, and add developers to the team. 
+
+## Enable RBAC in portal
+
+RBAC is disabled by default in the {{site.konnect_short_name}} portal. To enable RBAC, you must make a `PATCH` request to the portal configuration endpoint. The following example shows how to enable RBAC in the portal. For more details on how to create your personal access token, see [Authentication](/konnect/api/#authentication). 
+
+```bash
+curl --request PATCH \
+  --url https://<region>.api.konghq.com/konnect-api/api/portals/<portal-id> \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "rbac_enabled": true
+  }'
+```
+
+{:.note}
+> The `portal-id` can be found in the {{site.konnect_short_name}} Portal within the **Developers** tab. 
```

Comment:
The current wording is a little confusing to me. 

<img width="784" alt="image" src="https://user-images.githubusercontent.com/5304468/216727689-5aba7187-3d5a-4cf8-bba7-d2f643debca0.png">


`{{site.konnect_short_name}}` is replaced with `Konnect Portal`, but I think it should just be `Konnect`. Also, the portal id has been added to all subsections of the Dev Portal Section of Konnect, not just the Developers Tab (I believe this is recent enhancement)


I think it should be something like: 
```
The `portal-id` can be found in the upper right corner of the **Dev Portal** section in Konnect.
```




<img width="494" alt="image" src="https://user-images.githubusercontent.com/5304468/216727631-2402cd4b-4a55-46c3-b85c-98e41563768a.png">